app/page.tsx

```
@@ -137,10 +137,10 @@ export default async function Home() {
                 <div className="youtube-container">
                   <div className="preload-overlay"></div>
                   <iframe
-                    src={portfolioData.videoUrl.includes('youtube.com') ?
-                      `${portfolioData.videoUrl.replace('watch?v=', 'embed/')}?autoplay=1&mute=1&loop=1&playlist=${portfolioData.videoUrl.split('v=')[1]}&controls=0&showinfo=0&rel=0&modestbranding=1&iv_load_policy=3&enablejsapi=1&playsinline=1` :
-                      portfolioData.videoUrl.includes('youtu.be') ?
-                        `${portfolioData.videoUrl.replace('youtu.be/', 'youtube.com/embed/')}?autoplay=1&mute=1&loop=1&playlist=${portfolioData.videoUrl.split('youtu.be/')[1]}&controls=0&showinfo=0&rel=0&modestbranding=1&iv_load_policy=3&enablejsapi=1&playsinline=1` :
+                    src={portfolioData.videoUrl && portfolioData.videoUrl.includes('youtube.com') ?
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)